vql/networking/http_client.go

```diff
@@ -342,6 +350,14 @@ func (self *_HttpPlugin) Call(
 			return
 		}
 
+		// Validate HTTP Method
+		arg.Method = strings.ToUpper(arg.Method)
+		if _, ok := validHttpMethods[arg.Method]; !ok {
```

Comment:
This is unnecessary now since we have a switch below anyway - might as well just refactor this check into the switch